validation/markets-validator.js

```diff
@@ -0,0 +1,549 @@
+#!/usr/bin/env node
```

Comment:
There isn't a single comment in this whole file :wink: 

Please document *at least* the calling conventions of your functions.